app/routes.py

```diff
@@ -1,2 +1,284 @@
+        video.total_inventory = form_data["total_inventory"]
+
+    
+        db.session.commit()
+        
+        return jsonify(video.to_json())
+
+    elif request.method == "DELETE":
+        db.session.delete(video)
+        db.session.commit()
+        
+        return jsonify (
+        {
+            "id": video.id 
+        })   
+
+        
+    
+@video_bp.route("", methods=["POST"])
+def create_a_video(): 
+    request_body = request.get_json()
+
+    for video_attribute in ["title", "total_inventory", "release_date"]: 
+        if not (video_attribute in request_body): 
+            return jsonify({}), 400 
+
+    new_video = Video.from_json(request_body)  
+    db.session.add(new_video)
+    db.session.commit()
+
+
+    response = {
+             "id": new_video.id 
+
+               }
+    return make_response(jsonify(response), 201)
+        
+   
+
+
+# #Routes for Rental 
+@rental_bp.route("/check-out", methods=["POST"])
+def rentals_check_out(): 
+    request_body = request.get_json()
+
+    for rental_attribute in ["customer_id", "video_id"]: 
+        val= request_body[rental_attribute]
+        if not (rental_attribute in request_body) or not isinstance(val,int): 
+            return jsonify({}), 400
+
+    customer = Customer.query.filter_by(id=request_body["customer_id"]).first()
+    video = Video.query.filter_by(id=request_body["video_id"]).first()
+
+    if customer is None or video is None:
+        return jsonify(""), 404
+
+    if video.available_inventory == 0: 
+        return jsonify(""), 400
+
+    new_check_out = Rental(video_id = request_body["video_id"],
+                           customer_id = request_body ["customer_id"],
+                           due_date = datetime.now() + timedelta(days=7)
+    )
+
+    
+    customer.videos_checked_out_count += 1 
+    video.available_inventory -= 1
+    
+    
+    db.session.add(new_check_out)
+    db.session.commit()
+
+
+    response = {
+                "customer_id": request_body["customer_id"],
+                "video_id": request_body["video_id"],
+                "due_date": new_check_out.due_date,
+                "videos_checked_out_count": customer.videos_checked_out_count,
+                "available_inventory": video.available_inventory
+            }
+
+    return jsonify(response), 200
+    
+
+
+@rental_bp.route("/check-in", methods=["POST"])
+def rentals_check_in(): 
+    request_body = request.get_json()
+
+  
+    for rental_attribute in ["customer_id", "video_id"]: 
+        val= request_body[rental_attribute]
+        if not (rental_attribute in request_body) or not isinstance(val,int): 
+            return jsonify({}), 400
+        
+    # foo = ["foo"]
+    # foo["foo"]
+
+    customer = Customer.query.filter_by(id=request_body["customer_id"]).first()
+    video = Video.query.filter_by(id=request_body["video_id"]).first()
+    rental = Rental.query.filter_by(video_id=request_body["video_id"],customer_id=request_body["customer_id"]).first()
```

Comment:
Just another note that you do a lot of queries like this and they would make great methods to put in the model, maybe as class methods. 

That way you would separate the database stuff from handling the requests-responses.